include/deal.II/base/array_view.h

```diff
@@ -48,15 +48,19 @@ DEAL_II_NAMESPACE_OPEN
  * template argument type of this class needs to be @p const as well. The
  * following code snippet gives an example:
  * @code
- *   std::vector<int>       array       = get_data();  // a writable array
- *
- *   ArrayView<int> view (&array[5], 5);               // a view of
- * elements 5..9 (inclusive) view[2] = 42; // array[7] is set to 42
- *
- *   ArrayView<const int> const_view (&array[5], 5);   // same view, but
- * read-only int element_7 = const_view[2];                    // returns 42
- *   const_view[2] = 42;                               // error, can't write
- * into this view
+ * // a writable array
+ * std::vector<int> array = get_data();
+ *
+ * // a view of elements 5..9 (inclusive)
+ * ArrayView<int> view (&array[5], 5);
+ * view[2] = 42; // array[7] is set to 42
+ *
+ * // same view, but read-only
+ * ArrayView<const int> const_view (&array[5], 5);
+ * // returns 42
+ * int element_7 = const_view[2];
```

Comment:
can you attach the `returns 42` to the line in question? it's not clear the way it looks right now whether that's for the following or previous line.

Reply:
@drwells ping

Reply:
Or maybe better change the comment to "set element_7 to 42", but indeed after the statement in question